boards/DCM/Src/App/App_DcmWorld.c

```diff
@@ -24,9 +26,12 @@ struct DcmWorld *App_DcmWorld_Create(
     struct BrakeLight *const        brake_light,
     struct Buzzer *const            buzzer,
     struct ErrorTable *const        error_table,
-    struct Clock *const             clock)
+    struct Clock *const             clock,
+
+    bool (*is_buzzer_on)(struct DcmWorld *),
+    void (*buzzer_complete_callback)(struct DcmWorld *))
 {
-    struct DcmWorld *world = (struct DcmWorld *)malloc(sizeof(struct DcmWorld));
+    struct DcmWorld *world = malloc(sizeof(struct DcmWorld));
```

Comment:
Got rid of casting here https://stackoverflow.com/questions/605845/do-i-cast-the-result-of-malloc